app/views/layouts/application.html.erb

```diff
@@ -1,21 +1,26 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js" lang="en">
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Chef Identity Service</title>
-  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= stylesheet_link_tag "application", media: "all" %>
+  <%= javascript_include_tag '//cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js' %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-  <header class="navbar navbar-default" role="navigation">
-    <div class="container">
-      <div class="navbar-header">
-        <%= link_to root_path, :id => "logo", :class => "navbar-brand logo" do %>
-          <%= image_tag 'logo.svg' %>
-        <% end %>
-      </div>
-      <div class="collapse navbar-collapse">
-        <ul class="nav navbar-nav navbar-right">
+  <div class="contain-to-grid">
+    <nav class="top-bar" data-topbar role="navigation">
+      <ul class="title-area">
+        <li class="name">
+          <h1>
+            <%= link_to '', root_path, :class => 'logo', :data => {'tag-line' => 'oc-id'} %>
```

Comment:
We should lose the `tag-line` here. Maybe just class `logo default` until we figure out more around branding.